src/frontend/game/ClientContextProvider.tsx

```diff
@@ -94,9 +94,14 @@ const ClientContextProvider: FunctionalComponent = ({ children }) => {
 		[clientContext],
 	);
 
+	// we know we've synced with the server when we can find our
```

Comment:
if you're using this pr to learn how to create a new game event this file can be ignored, the changes are unrelated to implementing the ChangePlayerNameEvent.